modules/disable-sr-iov-subscription-update.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/configuring-unsupported-nic-sriov-operator.adoc
+
+:_content-type: PROCEDURE
+[id="disable-sr-iov-subscription-update_{context}"]
+= Enabling SR-IOV Network Operator developer mode
+
+Follow this procedure to enable SR-IOV Network Operator developer mode to allow the use of an unsupported NIC.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* You must have installed the SR-IOV Network Operator.
+
+.Procedure
+
+. Find the name of the Subscription CR for the SR-IOV Network Operator:
++
+[source,terminal]
+----
+$ oc get subscription -n openshift-sriov-network-operator
+----
+
+. Update the subscription by patching its spec field by using the following command:
++
+[source,terminal]
+----
+$ oc patch subscription sriov-network-operator -n openshift-sriov-network-operator --type=merge -p '{"spec": {"config": {"env": [{"name": "DEV_MODE", "value": "TRUE"}]}}}'
```

Comment:
in the previous docs the subscription name is `sriov-network-operator-subscription`,  so better update to 
`s/sriov-network-operator/sriov-network-operator-subscription`